DZ_Shabalina1.xml

```diff
@@ -0,0 +1,12 @@
+<Primary_School_Student>
+<name>Andrew</name>
+<age>8</age>
+<grade>2</grade>
+<average_score>4.6</average_score>
+<attends_after_school_program>true</attends_after_school_program>
+<favourite_lessons>
+<favourite_lesson>English</favourite_lesson>
```

Comment:
It's still valid although additional indentations would help to visualize the hierarchy, we'll learn how to do it automatically a bit later :)


![image](https://github.com/user-attachments/assets/894c3841-ec01-4901-8a3f-ef5ec29ac934)